playbooks/roles/rsyslog/tasks/main.yml

```diff
@@ -1,13 +1,20 @@
 ---
-- name: Copy the modified rsyslog configuration into place that prevents Libreswan and OpenConnect traffic from being logged
+- name: Copy the modified rsyslog configuration into place that prevents Libreswan traffic from being logged
   copy:
-    src: "{{ item }}"
-    dest: /etc/rsyslog.d/{{ item }}
+    src: 40-ipsec.conf
+    dest: /etc/rsyslog.d/40-ipsec.conf
     owner: root
     group: root
     mode: 0644
-  with_items:
-    - 40-ipsec.conf
-    - 41-openconnect.conf
+  notify: Restart rsyslog
+
+- name: Copy the modified rsyslog configuration into place that prevents OpenConnect traffic from being logged
+  copy:
+    src: 41-openconnect.conf
+    dest: /etc/rsyslog.d/40-ipsec.conf
```

Comment:
this is the same `dest` as from the copy task above & will stomp the `40-ipsec.conf` file. I think you want `41-openconnect.conf` here instead.